tests/cypress/tests/server/copy_paste_test.ts

```diff
@@ -14,19 +15,11 @@ describe('copy paste', () => {
 
         DOM.focused.type(`${Key.Meta}c`);
         DashTable.getCell(3, 0).click();
-
-	/**
-	 * Wait for cell to become editable (see
-	 * https://github.com/plotly/dash-table/issues/577).
-	 * The get('input') below is used for any case
-	 * in which a cell is being edited.
-	 */
-        DashTable.getCell(3, 0).get('input');
         DOM.focused.type(`${Key.Meta}v`);
-        DashTable.getCell(0, 0).click();
+        DashTable.getCell(0, 0, State.Any).click();
 
         for (let row = 0; row <= 2; ++row) {
-            DashTable.getCell(row + 3, 0).within(() => cy.get('.dash-cell-value').should('have.html', `${row}`));
+            DashTable.getCell(row + 3, 0, State.Any).within(() => cy.get('.dash-cell-value').should('have.html', `${row}`));
```

Comment:
This test does not require the table to be ready or loading, so it just gets whatever there is.